lib/core/backend/backend.ts

```diff
@@ -567,6 +567,28 @@ export class Backend {
       throw runtimeError.get('already_started', 'start');
     }
 
+    // When the support of cookie as jwt is enabled
+    // we need to perform some checks before starting Kuzzle
+    if (this.config.content.security.supportBrowserCookieAsJwt) {
+      if (this.config.content.http.accessControlAllowOrigin === '*') {
+        if (process.env.NODE_ENV === 'development') {
```

Comment:
`process.env` calls should be avoided in the code since each call will lead to an expensive syscall to get the env.

You can use our custom `global.NODE_ENV` variable which is set at startup.
```suggestion
        if (global.NODE_ENV === 'development') {
```

Also I think we should stop Kuzzle startup sequence even in production environment instead of (almost) silently start Kuzzle without a key security feature